cfgov/cfgov/settings/base.py

```diff
@@ -794,3 +798,15 @@
         'TIMEOUT': None,
     }
 }
+
+# Set our CORS allowed origins based on a JSON list in the
+# CORS_ALLOWED_ORIGINS environment variable.
+try:
+    CORS_ALLOWED_ORIGINS = json.loads(
+        os.environ.get("CORS_ALLOWED_ORIGINS", "[]")
+    )
+except (TypeError, ValueError):
+    raise ImproperlyConfigured(
+        "Environment variable CORS_ALLOWED_ORIGINS is either not defined or "
```

Comment:
> either not defined

If the variable isn't defined, this exception won't happen, will it, since the value will default to `"[]"`?

Reply:
Correct. The `ImproperlyConfigured` error should only be raised if the variable is defined but not valid JSON.

Reply:
Right. I see. I'll fix the message.